### PR TITLE
Issue 328 - refactor allowed admins, validators & decrypters

### LINKED ISSUE #328: Refactor config getAuthorizedDecrypters, getAllowedValidators, getAllowedAdmins
All these 3 functions read an array of addresses. Only thing that changes its the ..env var name
code could be simplified and merged into 1 single more generic function

### PR DESCRIPTION
**Pull request by paulo-ocean**
Fixes #328 

Changes proposed in this PR:

- refactor similar functions (allowed validators, decrypters and admins)  into a generic one (reads array of addresses())
- fix printCurrentConfig format

### CODE REVIEW

File: src/utils/config.ts [hunk 1]
```diff
@@ -469,5 +434,5 @@ function configChanged(previous: OceanNodeConfig, current: OceanNodeConfig): boo
 // useful for debugging purposes
 export async function printCurrentConfig() {
   const conf = await getConfiguration(true)
-  console.log(JSON.stringify(conf), null, 4)
+  console.log(JSON.stringify(conf, null, 4))
```
Comment (jamiehewitt15) [inline src/utils/config.ts:437]: Is there any particular reason we're not using `CONFIG_LOGGER` here? I suggest we update it

Comment (paulo-ocean) [reply src/utils/config.ts:437]: depends, do you want to expose the configuration on the logs?

**Review by mariacarmina — APPROVED** [review]
lgtm

**Review by bogdanfazakas — APPROVED** [review]
lgtm

**Pull request merged**